packages/hardhat-zksync-deploy/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const PLUGIN_NAME = '@matterlabs/hardhat-zksync-deploy';
+export const ETH_DEFAULT_NETWORK_RPC_URL = 'http://localhost:8545';
```

Comment:
I'm not a fan of extracting constants into a separate file if there's no good reason for that, especially if you're doing this for just 2 of them (and not all others). If we use each of them in one file, we can declare them there. Less indirection is better IMHO.

Reply:
Well, it's true that these 2 are currently used in just one file, but the main reason why I extracted them into a separate file is the fact that this codebase will grow and in my opinion it is cleaner to keep generic constants like these at a separate location. Of course I can change this if you insist.

Reply:
Not a strong opinion, keep it if you prefer.